README.md

```diff
@@ -2,4 +2,21 @@
 
 [![Build Status](https://travis-ci.org/ContinuumIO/intake-xarray.svg?branch=master)](https://travis-ci.org/ContinuumIO/intake-xarray)
 
-Intake-netcdf: xarray Plugin for Intake
+Intake-xarray: xarray Plugin for [Intake](https://github.com/ContinuumIO/intake)
+
+See [Intake docs](https://intake.readthedocs.io/en/latest/overview.html).
+
+In Intake-xarray, there are plugins provided for reading data into xarray with.
+  - NetCDF
+  - Rasterio
+  - Zarr
+
+### Installation
+
+The conda install instructions are:
+
+```
+pip intall rasterio
+conda install -c conda-forge zarr
+conda install -c intake intake_xarray
```

Comment:
Actually, only the last one is *required*, the other two are required only if you mean to use the specific plugins which reference them.